public/main.js

```diff
@@ -16,3 +26,54 @@ $browse.addEventListener('change', function (event) {
 
   $picFile.value = $browse.value.substring(fileNameIndex + 1)
 })
+
+$save.addEventListener('click', function (event) {
```

Comment:
Use adding/removing a CSS class to show and hide instead of manipulating the `style` property directly.